runtime/src/lib.rs

```diff
@@ -617,6 +617,7 @@ parameter_types! {
     pub MinimumRedemption: u32 = 0;
     pub WithdrawalPeriod: <Runtime as frame_system::Config>::BlockNumber = 10;
     pub DOTContributionLimit: Balance = 999;
+    pub PalletIndexStringLimit: u32 = 8;
```

Comment:
This might be too low, bumping this to 50 should be fine
```suggestion
    pub PalletIndexStringLimit: u32 = 50;
```